Formula/dehomopolymerate.rb

```diff
@@ -0,0 +1,20 @@
+class Dehomopolymerate < Formula
+  desc "Pairwise SNP distance matrix from a FASTA sequence alignment"
```

Comment:
Should be `Collapse sequence homopolymers to a single character`

Reply:
thanks - cut+paste error :)